test/with-express-raw.test.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * @jest-environment node
+ */
+import { Server } from 'http'
+import { AddressInfo } from 'net'
+import { promisify } from 'util'
+import fetch from 'node-fetch'
+import express, { Express } from 'express'
+import { RequestHandler, rest } from 'msw'
+import { createMiddleware } from '../src'
+import { ServerApi } from '@open-draft/test-server'
+
+// we don't need SSL for this test
+let server: Omit<ServerApi, 'https'>
+
+const handlers: RequestHandler[] = [
+  rest.post('/user', async (req, res, ctx) => {
+    const { firstName } = await req.json()
+    return res(ctx.set('x-my-header', 'value'), ctx.json({ firstName }))
+  }),
+]
+
+beforeAll(async () => {
+  server = await createServer((app) => {
+    app
+      .use(express.raw({ type: () => true }))
+      .use(createMiddleware(...handlers))
+  })
+})
+
+afterAll(async () => {
+  await server.close()
+})
+
+it('supports usage alongside with the "express.raw()" middleware', async () => {
+  const res = await fetch(server.http.makeUrl('/user'), {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ firstName: 'John' }),
+  })
+
+  expect(res.status).toBe(200)
+  expect(res.headers.get('x-my-header')).toBe('value')
+  expect(await res.json()).toEqual({ firstName: 'John' })
+})
+
+/**
+ * The test server created by @open-draft/test-server automatically
```

Comment:
This is peculiar. I need to look into disabling/customizing the default middleware.